README.md

```diff
@@ -54,6 +54,15 @@ To install the binary run `make install`.
 
 **Note**: Requires [Go 1.13+](https://golang.org/dl/)
 
+## Running in docker
+
+- Open and fill `.env` file with all necessary data
```

Comment:
I'd rather see this as a `make` target that executes the script.

Reply:
So how you see entire flow here?

Reply:
`$ make docker`

```
docker:
  @sh scripts/start-docker.sh
```

Reply:
Sorry, did not understand you correctly firs time.
Done in this [commit](https://github.com/fissionlabsio/juno/pull/18/commits/3ceb7a62a01808724276cdffb778ed253b1358c1)